admin/templates/notifications/handle_duplicate_notifications.html

```diff
@@ -22,32 +22,51 @@ <h2>Duplicate Notifications</h2>
                 <thead>
                     <tr>
                         <th>Select</th>
-                        <th>User</th>
-                        <th>Node</th>
+                        <th>ID</th>
                         <th>Event Name</th>
                         <th>Created</th>
                         <th>Count</th>
+                        <th>Email Transactional</th>
+                        <th>Email Digest</th>
                     </tr>
                 </thead>
                 <tbody>
                 {% for notification in duplicates %}
                     <tr>
                         <td><input type="checkbox" name="selected_notifications" value="{{ notification.id }}"></td>
-                        <td>{{ notification.user }}</td>
-                        <td>{{ notification.node }}</td>
```

Comment:
Node would be a useful field to include for Product, if it was linked to the admin page for the node detail.

The M2Ms are useful for comparing to the contributors list, to help determine which duplicate is the better one to keep, so it would probably be even more useful if duplicate notification info was serialized on the node detail page instead of here. That approach would also mitigate the pagination concerns.

Reply:
"it would probably be even more useful if duplicate notification info was serialized on the node detail page instead of here"

I agree with this. Instead of a list of all duplicated subscriptions everywhere, let's focus on one node at a time, and put this as part of the Node panel in the admin. That will eliminate the slow database query and make it easier to use.